examples/go/vm-reconfig-via-tag/README.md

```diff
@@ -0,0 +1,146 @@
+# Installing and using VM-Reconfig-Via-Tag Function
```

Comment:
Nit: there's some nice line-wrap tools for IDEs/editors which force line-wrap after 80/120 chars (IIRC VSCode has it built-in).

Reply:
Also: can we have a short "About this function" section to understand the intent/use case please?

Reply:
Good catch on the line wrapping. I'll have those added.

Yup, "About this function" section will be nice. I'll add that.